android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -284,36 +288,69 @@ constructor(
   fun formatPopulationMeasureReport(
     measureReport: MeasureReport
   ): List<MeasureReportPopulationResult> {
-    return measureReport.group.flatMap { reportGroup: MeasureReport.MeasureReportGroupComponent ->
-      reportGroup.stratifier.map { stratifier ->
-        val resultItems: List<MeasureReportIndividualResult> =
-          stratifier.stratum.filter { it.hasValue() }.map { stratifierComponent ->
-            val text =
-              when {
-                stratifierComponent.value.hasText() -> stratifierComponent.value.text
-                stratifierComponent.value.hasCoding() ->
-                  stratifierComponent.value.coding.last().display
-                else -> ""
-              }
+    return measureReport
+      .also { Timber.w(it.encodeResourceToString()) }
+      .group
+      .flatMap { reportGroup: MeasureReport.MeasureReportGroupComponent ->
+        reportGroup.stratifier.map { stratifier ->
+          val resultItems: List<MeasureReportIndividualResult> =
+            stratifier.stratum.filter { it.hasValue() }.map { stratum ->
+              val text =
+                when {
+                  stratum.value.hasText() -> stratum.value.text
+                  stratum.value.hasCoding() -> stratum.value.coding.last().display
+                  else -> "N/A"
+                }
 
-            val numerator = stratifierComponent.findPopulation(NUMERATOR)?.count ?: 0
-            val denominator = stratifierComponent.findPopulation(DENOMINATOR)?.count ?: 0
+              val numerator = stratum.findPopulation(NUMERATOR)?.count ?: 0
+              val denominator = reportGroup.findPopulation(NUMERATOR)?.count ?: 0
+              val percentage =
+                numerator.toDouble().div(if (denominator == 0) 1 else denominator) * 100.0
```

Comment:
If the denominator is 0, can we display the percentage as "N/A"? Or does this have to be a number? Something like below but load N/A from translation, if it has to be a number we can us 0 when the denominator is 0
```suggestion
                "N/A" if (denominator == 0) else numerator.toDouble().div(denominator) * 100
```
I think 100 should be fine since the first operand is a double already